sample-graphs/README.md

```diff
@@ -0,0 +1,6 @@
+# SourceCred Sample Graphs
+
+This directory contains sample graphs for research purposes, along with a utility for updating them.
+Each file is the cred graph for a particular repository, named as OWNER_REPO.json.
+
+To update the graphs with the latest data from GitHub, just run `./update.sh`. It may take a few minutes.
```

Comment:
There’ll be a nice error if you don’t have a token in your env, right?

Reply:
Correct. (Tested it.)